web/packages/teleport/src/Bots/Add/GitHubActions/ConnectGitHub.tsx

```diff
@@ -0,0 +1,359 @@
+                          title="Remove Rule"
+                          onClick={() =>
+                            setRepoRules(
+                              repoRules.filter((r, index) => index !== i)
+                            )
+                          }
+                        >
+                          <Icons.Trash size="medium" />
+                        </ButtonIcon>
+                      )}
+                    </Flex>
+                    <FormItem>
+                      <Text mt="3">Full URL to Your Repository*</Text>
+                      <FieldInput
+                        disabled={isLoading}
+                        rule={requireValidRepository}
+                        label=" "
+                        placeholder="ex. https://github.com/gravitational/teleport"
+                        value={repoRules[i].repoAddress}
+                        onChange={e =>
+                          handleChange(i, 'repoAddress', e.target.value)
+                        }
+                      />
+                    </FormItem>
+
+                    <FormItem>
+                      <Flex>
+                        <Box width="100%">
+                          <Text>
+                            Git Ref <OptionalFieldText />
+                          </Text>
+                          <Input
+                            disabled={isLoading}
+                            label="Git Ref"
+                            placeholder="main"
+                            style={{ borderRadius: '4px 0 0 4px' }}
+                            value={repoRules[i].ref}
+                            onChange={e =>
+                              handleChange(i, 'ref', e.target.value)
+                            }
+                          />
+                        </Box>
+                        <Box minWidth="160px">
+                          <Text ml="1">Ref Type</Text>
+                          <RefTypeSelect>
+                            <ReactSelect
+                              disabled={isLoading}
+                              isMulti={false}
+                              value={repoRules[i].refType}
+                              onChange={o => handleChange(i, 'refType', o)}
+                              options={refTypeOptions}
+                              menuPlacement="auto"
+                              className="react-select-container"
+                              classNamePrefix="react-select"
+                            />
+                          </RefTypeSelect>
+                        </Box>
+                      </Flex>
+                    </FormItem>
+
+                    <FormItem>
+                      <Text>
+                        Name of the GitHub Actions Workflow
+                        <OptionalFieldText />
+                      </Text>
+                      <FieldInput
+                        disabled={isLoading}
+                        placeholder="ex. cd"
```

Comment:
if `ex.` means example, we want to use `e.g.` (but if not, what does it mean?)

Reply:
It looks like this in the designs, I think it means example :thinking:

Reply:
Talked to Kenny, we decided to remove the `ex` here. TY